### PR TITLE
[DPM-701] forward affiliate id to newgame trx

### PR DESCRIPTION
**Pull request by georgiypetrov**

### CODE REVIEW

**Comment by georgiypetrov** [comment]
depends on https://github.com/DaoCasino/platform-back/pull/70

File: auth/repository/postgres/user.go [hunk 1]
```diff
@@ -11,7 +12,9 @@ import (
 const (
 	selectUserCntByAccNameStmt = "SELECT count(*) FROM users WHERE account_name = $1"
 	selectUserByAccNameStmt    = "SELECT * FROM users WHERE account_name = $1"
+	selectAffIDByNameStmt      = "SELECT affiliate_id FROM affiliates WHERE account_name $1"
```
Comment (justefg) [inline auth/repository/postgres/user.go:15]: should be `account_name = $1`

File: auth/usecase/usecase_test.go [hunk 2]
```diff
@@ -29,12 +29,14 @@ func TestAuthFlow(t *testing.T) {
 		accountName = "user"
 		email       = "user@user.com"
 		suid, _     = uuid.NewRandom()
+		affiliateID = ""
```
Comment (justefg) [inline auth/usecase/usecase_test.go:32]: better some non-empty value, i.e `"affiliate_1"`

File: auth/repository/postgres/user.go (hunk 1 above)
Comment (georgiypetrov) [reply auth/repository/postgres/user.go:15]: fix in #70

File: auth/usecase/usecase_test.go (hunk 2 above)
Comment (georgiypetrov) [reply auth/usecase/usecase_test.go:32]: fix in #70

**Review by justefg — APPROVED** [review]

**Pull request merged**